docs/articles/index.md

```diff
@@ -0,0 +1,136 @@
+# Index
+
+[A](#a) · [B](#b) · [C](#c) · [D](#d) · [E](#e) · [F](#f) · [G](#g) · [H](#h) · [I](#i) · [J](#j) · [K](#k) · [L](#l) · [M](#m) · [N](#n) · [O](#o) · [P](#p) · [Q](#q) · [R](#r) · [S](#s) · [T](#t) · [U](#u) · [V](#v) · [W](#w) · [X](#x) · [Y](#y) · [Z](#z)
+
+---
+
+## A
+[Access Control](access-control.md)  
+[ACLs](access-control.md)  
+[API](api.md)  
+[Application Groups](application-groups.md)  
+[Applications](applications.md)  
```

Comment:
The intend was to converge to services (as a core concept) and explaining the difference between apps and pods there. In every generic part of the docs, we should use "service".

I think I renamed Apps to Applications because I felt like using an abbreviation is weird. We can very well use _Apps_ though, but then we should be consequent about it.

Reply:
Thoughts:
- `Applications` sounds to me as generic as `Services`. I think we should pick one and stay with it
- `Apps` is short for `ApplicationSpec`, not for `Application`. Same for `Pod`. We could stick to `ApplicationSpec/App` and `PodSpec/Pod` to disambiguate between _Application/Service_ as a concept and their actual specs.

Reply:
I object. From a Marathon API perspective, there's a huge difference between an app and a pod. I agree that both apps and services are generic names – that why I suggest to document this explicitly, by pointing people to the core concept of a generic **service**. In that article, we can explain the diff between apps and pods, and link further to dedicated articles for each.

I think the contention comes from mingling types and names. WDYT about
* Services (explaining) what a service is from a Marathon perspective
* AppDefinition (which in detail describes that type)
* PodDefinition (which in detail describes that type)
?

Btw, we don't have an ApplicationSpec, and App is short for AppDefinition :P

Reply:
> Btw, we don't have an ApplicationSpec, and App is short for AppDefinition :P

We do have an [ApplicationSpec](https://github.com/mesosphere/marathon/blob/master/plugin-interface/src/main/scala/mesosphere/marathon/plugin/RunSpec.scala#L43) and `AppDefinition` inherits from it 😛

Reply:
> I think the contention comes from mingling types and names. WDYT about

Exactly. And my main point is: the word _Application_ is too generic to reference a specific type `AppDefinition/ApplicationSpec/App`